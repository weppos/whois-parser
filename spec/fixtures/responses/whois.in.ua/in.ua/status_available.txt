% This is the SUNIC WHOIS server.
% The Whois is subject to Terms of use
% See https://hostmaster.ua/documents.php
%
% In the process of delegation of a domain name,
% the Registrant is an entity who uses and manages a certain domain name,
% and the Registrar is a business entity that provides the Registrant
% with the services necessary for the technical maintenance of the
% registration and operation of the domain name.
% For information on the Registrant's domain name, you should contact the Registrar.

u34jedzcq.in.ua - Domain name does not exist
