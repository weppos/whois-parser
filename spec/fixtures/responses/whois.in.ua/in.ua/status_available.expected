#status
  %s == :available

#available?
  %s == true

#registered?
  %s == false


#created_on
  %s == nil

#updated_on
  %s == nil

#expires_on
  %s == nil


#nameservers
  %s %CLASS{array}
  %s == []
